### PR TITLE
add Daniel to docs

### PR DESCRIPTION
**Pull request by 2color**
### Summary

Grant me maintainer access to the docs teams

### What else do we need to know?

**DRI:** myself

### Reviewer's Checklist

- [ ] It is clear where the request is coming from (if unsure, ask)
- [ ] All the automated checks passed
- [ ] The YAML changes reflect the summary of the request
- [ ] The Terraform plan posted as a comment reflects the summary of the request

### CODE REVIEW

**Review by willscott — APPROVED** [review]

File: github/libp2p.yml [hunk 1]
```diff
@@ -4804,6 +4804,13 @@ repositories:
     default_branch: main
     delete_branch_on_merge: false
     description: How to use libp2p pubsub in browsers
+    files:
+      .github/pull_request_template.md:
+        content: .github/js_pull_request_template.md
+      .github/workflows/semantic-pull-request.yml:
+        content: .github/workflows/semantic-pull-request.yml
+      .github/workflows/stale.yml:
+        content: .github/workflows/stale.yml
```
Comment (p-shahi) [inline github/libp2p.yml:4813]: Can these changes be done in a separate pr?

Comment (galargh) [reply github/libp2p.yml]: Longer explanation of why they appeared here can be found here: https://github.com/libp2p/github-mgmt/pull/198#discussion_r1440325493

Shorter, now that https://github.com/libp2p/github-mgmt/pull/197 has been merged, these changes will disappear from this PR after the rebase.
